### ISSUE TITLE
Missing files after composer install

### ISSUE THREAD

**Issue by haelbichalex** [ModestCoders/magento2-dockergento#20]
I get the following error after setting dockergento up and running `dockergento composer install`:

```
Warning: require(/var/www/html/vendor/composer/../symfony/polyfill-mbstring/bootstrap.php): failed to open stream: No such file or directory in /var/www/html/vendor/composer/autoload_real.php on line 70

Fatal error: require(): Failed opening required '/var/www/html/vendor/composer/../symfony/polyfill-mbstring/bootstrap.php' (include_path='/var/www/html/vendor/magento/zendframework1/library:.:/usr/local/lib/php') in /var/www/html/vendor/composer/autoload_real.php on line 70
```
Then to check if everything is installed correctly, I rerun `dockergento composer install`, but the command echos: 
```
First time setup: copying all files from container
 > removing vendor content: '././vendor/*'
```
After that my vendor folder inside the container is empty.

**Comment by jalogut** [comment]
Hi @haelbichalex 

I think there might be an error in the `docker-compose.yml` configuration. Could you please paste here the content of your file? Thanks

**Comment by jalogut** [comment]
Hi @haelbichalex 

It seems to be an error in the `dockergento setup` command if the project is not a git repository. We are working on a fix for that. In the meantime can you do the following?

* Open  docker-compose.yml and remove this line `  - ./:/var/www/html/:delegated`
* Then execute the following

```
dockergento rebuild
dockergento composer install
```

**Comment by jalogut** [comment]
@haelbichalex I just added a fix for that on version `1.3.2` of [magento2-dockergento-console](https://github.com/ModestCoders/magento2-dockergent).
Could you please pull the latest version and then execute `dockergento setup` again? This time the `docker-compose.yml` should not contain the line `- ./:/var/www/html/:delegated`

Let us know if that fixes the problem. Thanks

**Comment by haelbichalex** [comment]
@jalogut Works like a charm now 👍 Thanks for your efforts.
